lib/libvfio-user.c

```diff
@@ -878,8 +878,8 @@ exec_command(vfu_ctx_t *vfu_ctx, struct vfio_user_header *hdr, size_t size,
              bool *free_iovec_data)
```

Comment:
This function can be simplified. 
e.g Looks like there is no need to pass _iovecs,.